pkg/kubectl/cmd/describe.go

```diff
@@ -81,7 +82,7 @@ func NewCmdDescribe(f cmdutil.Factory, out, cmdErr io.Writer) *cobra.Command {
 
 	cmd := &cobra.Command{
 		Use:     "describe (-f FILENAME | TYPE [NAME_PREFIX | -l label] | TYPE/NAME)",
-		Short:   i18n.T("Show details of a specific resource or group of resources"),
+		Short:   i18n.T("Show details of a specific resource or group of resources, include the uninitialized objects by default, unless user explicitly set --include-uninitialized=false"),
```

Comment:
IMO there is no need to include that info in short help text. And it may also cause problem to i18n.